topbeat/etc/topbeat.template.json

```diff
@@ -36,6 +36,22 @@
             }
           }
         },
+        "cpus": {
```

Comment:
Yes, it wasn't up to date. The changes must be in the template. Thanks for adding them.